docs/operations/recovery-from-permanent-quorum-loss-in-etcd-cluster.md

```diff
@@ -1,119 +1,130 @@
+If permanent quorum loss occurs to a multinode Etcd cluster, the operator needs to note down the PVCs, configmaps, statefulsets, CRs, etc. related to that Etcd cluster and work on those resources only. Following steps guide a human operator to recover from permanent quorum loss of an etcd cluster. We assume the name of the Etcd CR for the Etcd cluster is `etcd-main`.
 
 **Etcd cluster in shoot control plane of gardener deployment:**
-There are two [Etcd clusters](https://github.com/gardener/etcd-druid/tree/master/docs/proposals/multi-node) running in shoot control plane. One is named as `etcd-events` and another is named `etcd-main`. The operator needs to take care of permanent quorum loss to a specific cluster. If permanent quorum loss occurs to `etcd-events` cluster, the operator needs to note down the PVCs, configmaps, statefulsets, CRs etc related to `etcd-events` cluster and work on those resources only.
+There are two [Etcd clusters](https://github.com/gardener/etcd-druid/tree/master/docs/proposals/multi-node) running in shoot control plane. One is named as `etcd-events` and another is named `etcd-main`. The operator needs to take care of permanent quorum loss to a specific cluster. If permanent quorum loss occurs to `etcd-events` cluster, the operator needs to note down the PVCs, configmaps, statefulsets, CRs, etc. related to `etcd-events` cluster and work on those resources only.
 
 :warning: **Note:** Please note that manually restoring etcd can result in data loss. This guide is the last resort to bring an Etcd cluster up and running again.
 
-   If etcd-druid and etcd-backup-restore is being used with gardener, then
+If etcd-druid and etcd-backup-restore is being used with gardener, then
 
-   Target the control plane of affected shoot cluster via `kubectl`. Alternatively, you can use [gardenctl](https://github.com/gardener/gardenctl-v2) to target the control plane of the affected shoot cluster. You can get the details to target the control plane from the Access tile in the shoot cluster details page on the Gardener dashboard. Ensure that you are targeting the correct namespace.
+Target the control plane of affected shoot cluster via `kubectl`. Alternatively, you can use [gardenctl](https://github.com/gardener/gardenctl-v2) to target the control plane of the affected shoot cluster. You can get the details to target the control plane from the Access tile in the shoot cluster details page on the Gardener dashboard. Ensure that you are targeting the correct namespace.
 
-   1. Add the following annotation to the `Etcd` resource `kubectl annotate etcd etcd-main druid.gardener.cloud/ignore-reconciliation="true"`
+1. Add the following annotations to the `Etcd` resource `etcd-main`:
+    1. `kubectl annotate etcd etcd-main druid.gardener.cloud/suspend-etcd-spec-reconcile="true"`
+    
+    2. `kubectl annotate etcd etcd-main druid.gardener.cloud/resource-protection="false"`
```

Comment:
Let us stick with the convention as per https://kubernetes.io/docs/reference/kubectl/generated/kubectl_annotate/ and just use single quotes. I will make the changes to this doc

Reply:
The discussion of potentially removing the values of annotations being specified to the consumer in https://github.com/gardener/etcd-druid/pull/777#discussion_r1591861867 would supersede this discussion.

Reply:
@renormalize we have now changed the commands to simply add the annotation, with an empty value, since the code only requires the annotation key to be present, and doesn't really care about the value anymore.

Reply:
Thanks!